### ISSUE TITLE
GraphQL for Models with Timestamps Fails when created_at Is Requested

### ISSUE THREAD

**Issue by Buffalom** [duncanmcclean/runway#252]
### Description

I have enabled GraphQL for a model with timestamps enabled. When I request `created_at` using GraphQL the exception `Call to undefined method App\\Models\\Game::resolveGqlValue()` is thrown.

### Steps to reproduce

1. Add a migration for a model with timestamps enabled.
```php
Schema::create('games', function (Blueprint $table) {
    $table->id();
    $table->timestamps();
});
```

2. Create the model using `php artisan make:model Game`.

3. Add the resource with GraphQL enabled:
```php
'resources' => [
     \App\Models\Game::class => [
         'name' => 'Games',
         'blueprint' => 'game',
         'graphql' => true,
     ],
],
```

4. Generate the blueprint using `php please runway:generate-blueprints`.

5. Create an entry at `/cp/runway/game/create`.

6. Navigate to `/cp/graphiql` and run the following query:
```graphql
query Games {
  games {
    data {
      id
      created_at
    }
  }
}
```

7. At this point you should get the following exception:

```
Call to undefined method App\\Models\\Game::resolveGqlValue()
```

**Note:**
The exception is not thrown if only the `id` field and not `created_at` is requested.

**Workaround:**
In my case I was able to fix the issue, by implementing `resolveGqlValue()` on my model using the following trait. However, I am unsure as to how good of a workaround this is.

```php
trait ResolvesModelValues
{
    use ResolvesValues;

    public function augmentedValue($key) {
        return $this->{$key};
    }
}

```

### Environment

Environment
Application Name: Sporttag
Laravel Version: 9.52.5
PHP Version: 8.2.3
Composer Version: 2.5.4
Environment: local
Debug Mode: ENABLED
URL: gibb-sporttag-cms.novu.io
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: redis
Database: mysql
Logs: single
Mail: log
Queue: redis
Session: redis

Statamic
Addons: 2
Antlers: regex
Stache Watcher: Enabled
Static Caching: Disabled
Version: 3.4.7 PRO

Statamic Addons
doublethreedigital/runway: 4.2.3
statamic/eloquent-driver: 1.2.0

**Comment by duncanmcclean** [comment]
What does your blueprint look like?

**Comment by duncanmcclean** [comment]
I've had to introduce a breaking change (adding a trait to your Eloquent model) to fix this bug so this will be fixed in the next release, v5.0 which will be tagged alongside Statamic 4.

I'm going to close this issue now but you'll receive a notification on this issue once the fix has been tagged.

**Comment by Buffalom** [comment]
Awesome. Thanks a lot for the quick fix.
